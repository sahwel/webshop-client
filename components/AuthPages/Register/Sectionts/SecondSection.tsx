import { faFlag } from '@fortawesome/free-regular-svg-icons';
import {
  faLocationDot,
  faCity,
  faRoad,
  faHome,
  faPlus,
} from '@fortawesome/free-solid-svg-icons';
import Input from '../../../common/Form/Input/Input';

const SecondSection = () => {
  return (
    <>
      <p>
        Please enter your address and select where youâ€™d like to get you
        packages delivered!
      </p>
      <Input type="text" placeholder="Country" icon={faFlag} />
      <Input type="text" placeholder="Zip Code" icon={faLocationDot} />
      <Input type="text" placeholder="City" icon={faCity} />
      <Input type="text" placeholder="Street" icon={faRoad} />
      <Input type="text" placeholder="House" icon={faHome} />
      <Input type="text" placeholder="Other" icon={faPlus} />
      <div className="flex justify-between">
        <label
          htmlFor="is-billing-same-as-normal-address"
          className="cursor-pointer"
        >
          Billing address equals with your address.
        </label>
        <Input
          type="checkbox"
          id="is-billing-same-as-normal-address"
          value="is-billing-same-as-normal-address"
          name="is-billing-same-as-normal-address"
        />
      </div>
    </>
  );
};

export default SecondSection;
