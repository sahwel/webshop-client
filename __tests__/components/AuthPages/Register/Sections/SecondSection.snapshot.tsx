import { render, renderHook, RenderHookResult } from '@testing-library/react';
import SecondSection from '@/components/AuthPages/Register/Sectionts/SecondSection';

describe('SecondSection Snapshot', () => {
  it('renders SecondSection unchanged', () => {
    const { container } = render(<SecondSection />);
    expect(container).toMatchSnapshot();
  });
});
