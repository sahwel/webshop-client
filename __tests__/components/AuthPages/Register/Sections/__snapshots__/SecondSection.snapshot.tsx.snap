// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondSection Snapshot renders SecondSection unchanged 1`] = `
<div>
  <p>
    Please enter your address and select where youâ€™d like to get you packages delivered!
  </p>
  <div
    class="input_container"
  >
    <input
      class="input"
      placeholder="Country"
      type="text"
    />
    <svg
      aria-hidden="true"
      class="svg-inline--fa fa-flag input_icon icon-7"
      data-icon="flag"
      data-prefix="far"
      focusable="false"
      role="img"
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M476.3 0c-6.365 0-13.01 1.35-19.34 4.233c-45.69 20.86-79.56 27.94-107.8 27.94c-59.96 0-94.81-31.86-163.9-31.87c-34.63 0-77.87 8.003-137.2 32.05V24C48 10.75 37.25 0 24 0S0 10.75 0 24v464C0 501.3 10.75 512 24 512s24-10.75 24-24v-104c53.59-23.86 96.02-31.81 132.8-31.81c73.63 0 124.9 31.78 198.6 31.78c31.91 0 68.02-5.971 111.1-23.09C504.1 355.9 512 344.4 512 332.1V30.73C512 11.1 495.3 0 476.3 0zM464 319.8c-30.31 10.82-58.08 16.1-84.6 16.1c-30.8 0-58.31-7-87.44-14.41c-32.01-8.141-68.29-17.37-111.1-17.37c-42.35 0-85.99 9.09-132.8 27.73V84.14l18.03-7.301c47.39-19.2 86.38-28.54 119.2-28.54c28.24 .0039 49.12 6.711 73.31 14.48c25.38 8.148 54.13 17.39 90.58 17.39c35.43 0 72.24-8.496 114.9-26.61V319.8z"
        fill="currentColor"
      />
    </svg>
  </div>
  <div
    class="input_container"
  >
    <input
      class="input"
      placeholder="Zip Code"
      type="text"
    />
    <svg
      aria-hidden="true"
      class="svg-inline--fa fa-location-dot input_icon icon-7"
      data-icon="location-dot"
      data-prefix="fas"
      focusable="false"
      role="img"
      viewBox="0 0 384 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M168.3 499.2C116.1 435 0 279.4 0 192C0 85.96 85.96 0 192 0C298 0 384 85.96 384 192C384 279.4 267 435 215.7 499.2C203.4 514.5 180.6 514.5 168.3 499.2H168.3zM192 256C227.3 256 256 227.3 256 192C256 156.7 227.3 128 192 128C156.7 128 128 156.7 128 192C128 227.3 156.7 256 192 256z"
        fill="currentColor"
      />
    </svg>
  </div>
  <div
    class="input_container"
  >
    <input
      class="input"
      placeholder="City"
      type="text"
    />
    <svg
      aria-hidden="true"
      class="svg-inline--fa fa-city input_icon icon-7"
      data-icon="city"
      data-prefix="fas"
      focusable="false"
      role="img"
      viewBox="0 0 640 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M480 192H592C618.5 192 640 213.5 640 240V464C640 490.5 618.5 512 592 512H48C21.49 512 0 490.5 0 464V144C0 117.5 21.49 96 48 96H64V24C64 10.75 74.75 0 88 0C101.3 0 112 10.75 112 24V96H176V24C176 10.75 186.7 0 200 0C213.3 0 224 10.75 224 24V96H288V48C288 21.49 309.5 0 336 0H432C458.5 0 480 21.49 480 48V192zM576 368C576 359.2 568.8 352 560 352H528C519.2 352 512 359.2 512 368V400C512 408.8 519.2 416 528 416H560C568.8 416 576 408.8 576 400V368zM240 416C248.8 416 256 408.8 256 400V368C256 359.2 248.8 352 240 352H208C199.2 352 192 359.2 192 368V400C192 408.8 199.2 416 208 416H240zM128 368C128 359.2 120.8 352 112 352H80C71.16 352 64 359.2 64 368V400C64 408.8 71.16 416 80 416H112C120.8 416 128 408.8 128 400V368zM528 256C519.2 256 512 263.2 512 272V304C512 312.8 519.2 320 528 320H560C568.8 320 576 312.8 576 304V272C576 263.2 568.8 256 560 256H528zM256 176C256 167.2 248.8 160 240 160H208C199.2 160 192 167.2 192 176V208C192 216.8 199.2 224 208 224H240C248.8 224 256 216.8 256 208V176zM80 160C71.16 160 64 167.2 64 176V208C64 216.8 71.16 224 80 224H112C120.8 224 128 216.8 128 208V176C128 167.2 120.8 160 112 160H80zM256 272C256 263.2 248.8 256 240 256H208C199.2 256 192 263.2 192 272V304C192 312.8 199.2 320 208 320H240C248.8 320 256 312.8 256 304V272zM112 320C120.8 320 128 312.8 128 304V272C128 263.2 120.8 256 112 256H80C71.16 256 64 263.2 64 272V304C64 312.8 71.16 320 80 320H112zM416 272C416 263.2 408.8 256 400 256H368C359.2 256 352 263.2 352 272V304C352 312.8 359.2 320 368 320H400C408.8 320 416 312.8 416 304V272zM368 64C359.2 64 352 71.16 352 80V112C352 120.8 359.2 128 368 128H400C408.8 128 416 120.8 416 112V80C416 71.16 408.8 64 400 64H368zM416 176C416 167.2 408.8 160 400 160H368C359.2 160 352 167.2 352 176V208C352 216.8 359.2 224 368 224H400C408.8 224 416 216.8 416 208V176z"
        fill="currentColor"
      />
    </svg>
  </div>
  <div
    class="input_container"
  >
    <input
      class="input"
      placeholder="Street"
      type="text"
    />
    <svg
      aria-hidden="true"
      class="svg-inline--fa fa-road input_icon icon-7"
      data-icon="road"
      data-prefix="fas"
      focusable="false"
      role="img"
      viewBox="0 0 576 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M256 96C256 113.7 270.3 128 288 128C305.7 128 320 113.7 320 96V32H394.8C421.9 32 446 49.08 455.1 74.63L572.9 407.2C574.9 413 576 419.2 576 425.4C576 455.5 551.5 480 521.4 480H320V416C320 398.3 305.7 384 288 384C270.3 384 256 398.3 256 416V480H54.61C24.45 480 0 455.5 0 425.4C0 419.2 1.06 413 3.133 407.2L120.9 74.63C129.1 49.08 154.1 32 181.2 32H255.1L256 96zM320 224C320 206.3 305.7 192 288 192C270.3 192 256 206.3 256 224V288C256 305.7 270.3 320 288 320C305.7 320 320 305.7 320 288V224z"
        fill="currentColor"
      />
    </svg>
  </div>
  <div
    class="input_container"
  >
    <input
      class="input"
      placeholder="House"
      type="text"
    />
    <svg
      aria-hidden="true"
      class="svg-inline--fa fa-house input_icon icon-7"
      data-icon="house"
      data-prefix="fas"
      focusable="false"
      role="img"
      viewBox="0 0 576 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M575.8 255.5C575.8 273.5 560.8 287.6 543.8 287.6H511.8L512.5 447.7C512.5 450.5 512.3 453.1 512 455.8V472C512 494.1 494.1 512 472 512H456C454.9 512 453.8 511.1 452.7 511.9C451.3 511.1 449.9 512 448.5 512H392C369.9 512 352 494.1 352 472V384C352 366.3 337.7 352 320 352H256C238.3 352 224 366.3 224 384V472C224 494.1 206.1 512 184 512H128.1C126.6 512 125.1 511.9 123.6 511.8C122.4 511.9 121.2 512 120 512H104C81.91 512 64 494.1 64 472V360C64 359.1 64.03 358.1 64.09 357.2V287.6H32.05C14.02 287.6 0 273.5 0 255.5C0 246.5 3.004 238.5 10.01 231.5L266.4 8.016C273.4 1.002 281.4 0 288.4 0C295.4 0 303.4 2.004 309.5 7.014L564.8 231.5C572.8 238.5 576.9 246.5 575.8 255.5L575.8 255.5z"
        fill="currentColor"
      />
    </svg>
  </div>
  <div
    class="input_container"
  >
    <input
      class="input"
      placeholder="Other"
      type="text"
    />
    <svg
      aria-hidden="true"
      class="svg-inline--fa fa-plus input_icon icon-7"
      data-icon="plus"
      data-prefix="fas"
      focusable="false"
      role="img"
      viewBox="0 0 448 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M432 256c0 17.69-14.33 32.01-32 32.01H256v144c0 17.69-14.33 31.99-32 31.99s-32-14.3-32-31.99v-144H48c-17.67 0-32-14.32-32-32.01s14.33-31.99 32-31.99H192v-144c0-17.69 14.33-32.01 32-32.01s32 14.32 32 32.01v144h144C417.7 224 432 238.3 432 256z"
        fill="currentColor"
      />
    </svg>
  </div>
  <div
    class="flex justify-between"
  >
    <label
      class="cursor-pointer"
      for="is-billing-same-as-normal-address"
    >
      Billing address equals with your address.
    </label>
    <div
      class="input_container"
    >
      <input
        class="checkbox"
        id="is-billing-same-as-normal-address"
        name="is-billing-same-as-normal-address"
        type="checkbox"
        value="is-billing-same-as-normal-address"
      />
    </div>
  </div>
</div>
`;
